.github/workflows/deployProd.yml

```diff
@@ -1,10 +1,7 @@
 name: Deploy Prod
 
 on:
-  workflow_run:
-    workflows: [ "Deploy Stg" ]
-    types:
-      - completed
+  workflow_call:
```

Comment:
is this intentionally added here and not set to anything? 🤔

Reply:
Yes that is intentionally blank to just allow it to be used in a workflow_call, but Alis and I talked about this further and we changed our mind on this approach of using the workflow call since it could risk the prod deployment jobs getting lost in the deployStg workflow so I'll be pushing commits that revert that change